src/pybind/mgr/dashboard/frontend/src/app/ceph/block/mirroring/bootstrap-create-modal/bootstrap-create-modal.component.ts

```diff
@@ -71,18 +72,24 @@ export class BootstrapCreateModalComponent extends BaseModal implements OnDestro
         const poolName = pool['name'];
         const mirroring_disabled = pool['mirror_mode'] === 'disabled';
         const control = poolsControl.controls[poolName];
-        if (control) {
-          if (mirroring_disabled && control.disabled) {
-            control.enable();
-          } else if (!mirroring_disabled && control.enabled) {
+        if (mirroring_disabled || pool.peer_uuids.length > 0) {
```

Comment:
As far as I see `peer_uuids` is only used here, this is to verify that the pool exists, right? Any other reason?
If that's the case, is not enough to check the pool's name or the control. as was being done before?